### ISSUE TITLE
only i3 launch

### ISSUE THREAD

**Issue by x0rld** [i3-gnome/i3-gnome#69]
i'm on fedora 32 with gnome 3.36
when i choose I3 + gnome there is only I3 launch

**Comment by jcstr** [comment]
Hi!
It looks like you're getting started with i3-gnome.

I invite you to read the [wiki](https://github.com/i3-gnome/i3-gnome/wiki) and the [FAQ](https://github.com/i3-gnome/i3-gnome/issues?q=label%3AFAQ+is%3Aclosed) section from the issues.

In general `i3-gnome` can help to keep all of your sessions from Google Chrome, Firefox and so on on to **i3**. Also can help to keep all of the GTK themes and more components from **GNOME**.
 
TL;DR
`i3-gnome` is a friendly bridge that keep your stuff from **GNOME** to have it over i3.

**Comment by jcstr** [comment]
Closing due lack of activity.
